test/test_prototype_transforms.py

```diff
@@ -0,0 +1,184 @@
+import itertools
```

Comment:
These test are just a quick and dirty implementation to see if the transforms don't error when called with the supported inputs.